Smartling.API/Authentication/AuthApiUser.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Newtonsoft.Json;
+
+namespace Smartling.Api.Authentication
+{
+    public class AuthApiUser : AuthApiClient
+    {
+        private const string AUTH_API_V2_AUTHENTICATE_USER = "/auth-api/v2/authenticate/user";
+        private const string AUTH_API_V2_REFRESH_USER = "/auth-api/v2/authenticate/refresh/user";
```

Comment:
Did you try to refresh token with this url? I think refresh url should not be updated.

Reply:
Tested. I've got "The remote server returned an error: (401) Unauthorized."

Reply:
Yes, you are right